connectivity/netsocket/tests/TESTS/netsocket/udp/udpsocket_echotest_burst.cpp

```diff
@@ -21,7 +21,9 @@
 #include "unity/unity.h"
 #include "utest.h"
 #include "udp_tests.h"
+#ifdef MBED_CONF_APP_BAUD_RATE
 #include "CellularDevice.h"
+#endif // MBED_CONF_CELLULAR_PRESENT
```

Comment:
the comment states something different, is this app baudrate rather than cellular present?

Reply:
Just add the same ifdef as it is already doen bellow in this file